plugin-animation/src/test/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImplTest.kt

```diff
@@ -979,6 +990,16 @@ class CameraAnimationsPluginImplTest {
       duration = animatorDuration
     }
 
+  private fun CameraOptions.toCameraState(): CameraState {
+    return CameraState(
+      center ?: cameraOptions.center!!,
+      padding ?: cameraOptions.padding!!,
+      zoom ?: cameraOptions.zoom!!,
+      bearing ?: cameraOptions.bearing!!,
+      pitch ?: cameraOptions.pitch!!
+    )
+  }
+
```

Comment:
Would this extension function be worth exposing publicly?

Reply:
emm. I think this extension function relies on the local variable `cameraOptions`, wouldn't be applicable for universal use cases.